### PR TITLE
SingleStat panel class added

### PR DESCRIPTION
**Pull request by dmajere**
Here is a singlestat class

### CODE REVIEW

File: grafanalib/core.py [hunk 1]
```diff
@@ -646,3 +694,151 @@ def to_json_data(self):
...
+            'maxValue': self.maxValue,
+            'minValue': self.minValue,
+            'show': self.show,
+            'thresholdLabels': self.thresholdLabels,
+            'thresholdMarkers': self.thresholdMarkers,
+        }
+
+
+@attr.s
+class SingleStat(object):
```
Comment (jml) [inline grafanalib/core.py]: Could you please add a docstring for this?

**Comment by jml** [comment]
Lovely, thanks. Could you please add a changelog entry?

File: grafanalib/core.py (hunk 1 above)
Comment (dmajere) [inline grafanalib/core.py]: Done

**Comment by dmajere** [comment]
what version it will be? 0.1.3?

**Comment by jml** [comment]
> what version it will be? 0.1.3?

Just call it `NEXT` and we'll roll dice the next time we do a release  :)

**Comment by jml** [comment]
Actually, I just pushed a section for this in 1c72a4dd80da001c20e7deb471db352dce7b5317

**Comment by dmajere** [comment]
Done

**Comment by dmajere** [comment]
I'll do the same for other two

**Pull request merged**